### PR TITLE
Modify acoount activation email template

### PR DESCRIPTION
**Pull request by tanyaxp**
Modify account activation email text. This improves the user experience for auto
enrolled participants.

### CODE REVIEW

**Comment by tanyaxp** [comment]
@stvstnfrd , @caesar2164 Could you please review this. Thanks!

**Review by caesar2164 — DISMISSED** [review]

File: lagunita/lms/templates/emails/activation_email.txt [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<%! from django.utils.translation import ugettext as _ %>
```
Comment (stvstnfrd) [inline lagunita/lms/templates/emails/activation_email.txt:2]: It's minor, but to keep these two files as in-sync as possible,
let's also include the mako comment;
it makes `diff`ing more in-line.

```
## mako
```

**Review by stvstnfrd — CHANGES_REQUESTED** [review]

**Comment by stvstnfrd** [comment]
@tanyaxp This branch also needs rebased now that the other PR merged to master.
Feel free to perform the rebase now (instead of waiting until "the end" of the PR).

**Comment by tanyaxp** [comment]
@stvstnfrd Please take a look at this rebase. Thanks!

**Review by stvstnfrd — APPROVED** [review]

**Pull request merged**